recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "Copyright (c) 2003-2019 Gary P. Scavone"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["patches/*"]
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Linux":
+            self.requires("libalsa/1.2.4")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+            destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["RTMIDI_BUILD_TESTING"] = False
+        self._cmake.configure(source_folder=self._source_subfolder)
```

Comment:
You need to add a wrapper `CMakeLists.txt` file.
```suggestion
        self._cmake.configure()
```

Reply:
You need to manually add the wrapper cmake script.